src/ui/pages/RevealPrivateKey/RevealPrivateKey.tsx

```diff
@@ -426,7 +426,7 @@ function VerifySuccess({ seedType }: { seedType: SeedType }) {
   );
 }
 
-export function BackupWallet() {
+export function RevealPrivateKey() {
```

Comment:
This page is now exclusively for revealing the private key, which is why it has a new name